components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LoggingResource.java

```diff
@@ -61,7 +61,7 @@ public class LoggingResource extends APIResource {
     private JSONObject jsonBody;
     private static String filePath = System.getProperty(ServerConstants.CARBON_CONFIG_DIR_PATH) + File.separator
             + "log4j2.properties";
-    private static File logPropFile = new File(filePath);
+    private static final File logPropFile = new File(filePath);
```

Comment:
Shall we make `filePath` also a static variable and rename `filePath` and `logPropFile` as `FILE_PATH` and 
 `LOG_PROP_FILE` respectively?

Reply:
fixed the naming as suggested. the variables are already static. you might mean that it should be final as well right, since they are not being modifying elsewhere? making the variables "final" with the fix